cmd/root.go

```diff
@@ -26,11 +26,13 @@ const (
 )
 
 var (
-	ctx       Context
-	printer   Printer
-	detailer  Detailer
-	driverURL string
-	driver    *metalgo.Driver
+	ctx                   Context
+	printer               Printer
+	detailer              Detailer
+	driverURL             string
+	driver                *metalgo.Driver
+	defaultSSHPublicKeys  = [...]string{"id_ed25519.pub", "id_rsa.pub", "id_dsa.pub"}
```

Comment:
just specify the possibleKeyformats and add ".pub" for the public keys, so only one slice